crates/bin/pd/src/testnet/generate.rs

```diff
@@ -287,8 +287,8 @@ impl TestnetConfig {
             consensus_params: tendermint::consensus::Params {
                 abci: AbciParams::default(),
                 block: tendermint::block::Size {
-                    max_bytes: 22020096,
-                    max_gas: -1,
+                    max_bytes: 4194304,
+                    max_gas: 10_000_000,
```

Comment:
I don't think we want to set this, the penumbra sdk does not use the cometbft notion of gas at all, so i'm worried setting a limit here would clash with how people running the software expect comet to reap from the mempool.

Reply:
Yeah, although setting it to limitless (-1) also sort of sends the wrong message. Should it just to set to zero? That's a pretty clear self-documenting 'we don't use this' signal.

Reply:
I think it's best to leave it to `-1` unless we understand why it would be bad. As defined it reflects the reality that the application doesn't expect comet to suggest raw proposals based on gas metering. I think setting this to a concrete value will make comet actively use it in along with `max_bytes`. If that's the case, then a value of zero would make comet suggest empty raw proposals

Reply:
Appreciate the feedback, got rid of the change to max_gas. Also, turns out [the official docs](https://docs.cometbft.com/v0.38/spec/abci/abci++_app_requirements#states-and-abci-connections) themselves recommend -1 when 'the concept of gas is meaningless'!